scripts/core/get-superdesk-api-implementation.tsx

```diff
@@ -258,6 +258,7 @@ export function getSuperdeskApiImplementation(
         },
         privileges: {
             getOwnPrivileges: () => privileges.loaded.then(() => privileges.privileges),
+            hasPrivilege: (privilege: {[key: string]: number}) => privileges.userHasPrivileges(privilege),
```

Comment:
What's the schema for privileges? Can it only be `1` or `0`? If so, it'd be nicer to have the interface be `hasPrivilege(name: string): boolean`

Reply:
@tomaskikutis yes, the schema for privileges will have only 0 or 1,  for ex: `{mark_for_user: 1, mark_for_desk: 1, dashboard: 0}`